### DISCUSSION TITLE
Shield Pattern Fix

### DISCUSSION THREAD

**Discussion by Lexiosity** [BedrockTweaks/Files#405] [Ideas]
In Bedrock, the shield patterns are completely compressed for no reason, It should be same as banners, yet they aren't. It's not really difficult to fix, either. All you got to do is replace the current shield patterns with the banner patterns and make sure to follow the base.png format. If you need help, I do have the fixed patterns but I want everyone to be able to get them from BT. I also have banner diagonal fixes too.

![image](https://github.com/user-attachments/assets/aedbb45f-6326-4e58-bc02-7725545c67e2)

**Comment by PAPISOP** [comment]
You can use HD Shield Banner Patterns pack
